packages/core/snapshot/read.go

```diff
@@ -38,14 +39,22 @@ func streamSnapshotDataFrom(
 	header.LatestECRecord = ecRecord
 	headerConsumer(header)
 
+	// read solid entry points
+	for i := header.FullEpochIndex; i <= header.DiffEpochIndex; i++ {
+		seps, err := readSolidEntryPoints(scanner)
```

Comment:
It would be better not to use delimiters but length fields instead for the whole snapshot.

Reply:
Will fix that all in another PR.